core/src/main/java/org/aerogear/mobile/core/metrics/publisher/NetworkMetricsPublisher.java

```diff
@@ -34,7 +36,7 @@ public NetworkMetricsPublisher(final Context context, final HttpRequest httpRequ
     }
 
     @Override
-    public void publish(final Metrics... metrics) {
+    public void publish(final Metrics[] metrics, @Nullable final Callback callback) {
```

Comment:
Add `@nonNull` annotation

Reply:
According to the actual implementation, it is `@nullable`, because the publisher doesn't have another method without a callback

Reply:
Yes, I meant add `@NonNull` annotation to the `metrics` array